apps/meteor/app/lib/server/lib/notifyUsersOnMessage.ts

```diff
@@ -1,30 +1,22 @@
+import moment from 'moment';
+
+import { IMessage, IRoom, RoomType } from '@rocket.chat/core-typings';
```

Comment:
can't you
```typescript
import type { IMessage, IRoom, RoomType } from '@rocket.chat/core-typings';
```
??

Reply:
Sure! Added on https://github.com/RocketChat/Rocket.Chat/pull/31877/commits/60aac2cbb3801f5bb26fcc51b930dfde02156df9.